Tests/Resources/app/config/config.php

```diff
@@ -5,3 +5,10 @@
 
 $loader->import(CMF_TEST_CONFIG_DIR.'/default.php');
 $loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
+
+$config = array(
+    'form' => array(
+        'csrf_protection' => false,
+    ),
+);
+$container->loadFromExtension('framework', $config);
```

Comment:
did this and the csrf protection just pop up with the additional tests you do or is this a general fix for a problem?